### PR TITLE
Origin/feature/savebooking

### PR DESCRIPTION
**Pull request by manic1990**
Added booking view

### CODE REVIEW

File: BookingSystem/Controllers/CustomerController.cs [hunk 1]
```diff
@@ -58,8 +58,8 @@ public async Task<IActionResult> Index()
         public async Task<IActionResult> Customer(ViewCustomerData model)
         {
             var DataValue = _mapper.Map<CustomerVM>(model);
-            await _clientCustomerRepository.SaveCustomerAsync(DataValue);
-            return RedirectToAction("Index");
+            var result = await _clientCustomerRepository.SaveCustomerAsync(DataValue);
+            return RedirectToAction("Booking", "Booking");
```
Comment (vivekr2021) [inline BookingSystem/Controllers/CustomerController.cs:62]: Please avoid using the same names between the same solution. "Booking" Controller, "Booking" ActionResult

**Review by vivekr2021 — CHANGES_REQUESTED** [review]

File: BookingSystem/Controllers/BookingController.cs [hunk 2]
```diff
@@ -18,17 +19,28 @@ public class BookingController : Controller
...
+        private readonly IAPIClientRoomRepository _clientRoomRepository;
         private readonly IMapper _mapper;
         private readonly ILogger _logger;
         private const string start = " method started";
         private const string complete = " method completed";
         private const string GetBooking = "GetBookingAsync";
-        public BookingController(IAPIClientBookingRepository clientRepository, IAPIClientCustomerRepository clientCustomerRepository, IMapper mapper, ILoggerFactory logFactory)
+        public BookingController(IAPIClientBookingRepository clientRepository,
+            IAPIClientCustomerRepository clientCustomerRepository,
+            IMapper mapper, ILoggerFactory logFactory,
```
[the remark this answers is not in the document]
Comment (manic1990) [reply BookingSystem/Controllers/BookingController.cs]: Modified

File: BookingSystem/Controllers/CustomerController.cs (hunk 1 above)
Comment (manic1990) [reply BookingSystem/Controllers/CustomerController.cs]: Changed as nameof action name

**Review by Backiaraj07 — APPROVED** [review]
Code reviewed.

**Review by Backiaraj07 — APPROVED** [review]
Verfied

**Pull request merged**